### ISSUE TITLE
Borderless PDF annotations

### ISSUE THREAD

**Issue by MikhailRyazanov** [otfried/ipe#397]
It would be nice to have an option to disable the borders around PDF annotations generated by the `url` attribute of the `<group>` tag. According to the Adobe's “PDF Reference” book (section 8.4.1 in the 3rd edition), this can be done by adding
```
/Border [0 0 0]
```
to the annotation dictionary. Basically, appending it after this part
```cpp
      iStream << "<<\n"
	      << "/Type /Annot\n"
	      << "/Subtype /Link\n"
	      << "/H /N\n"
```
in `ipepdfwriter.cpp`.

**Comment by otfried** [comment]
Good point!  I don't think it should even be optional - one could always draw a rectangle in Ipe if one wants to highlight the link.

I will just remove the border.

**Comment by otfried** [comment]
Implemented in 7.2.25.
